README.md

```diff
@@ -213,6 +213,12 @@ The name the puppet agent daemon should run as.
 
 - *Default*: puppet
 
+ssldir
```

Comment:
What about defaulting to `undef` and if undef use `$vardir/ssl` ? This way if not undef, we can use `validate_absolute_path()`